compiler/src/main/java/com/squareup/anvil/compiler/ModuleMerger.kt

```diff
@@ -52,13 +54,26 @@ internal class ModuleMerger(
           ?.value
           ?.map { it.toType(codegen) }
 
+    val anvilModuleName = createAnvilModuleName(codegen.descriptor)
+
     val modules = classScanner
         .findContributedClasses(
             module = codegen.descriptor.module,
             packageName = HINT_CONTRIBUTES_PACKAGE_PREFIX,
             annotation = contributesToFqName
         )
         .asSequence()
+        .filter {
+          // We generate a Dagger module for each merged component. We use Anvil itself to
+          // contribute this generated module. It could happen that there are multiple components
+          // merging the same scope or different Gradle modules depend on each other and in each
+          // the same scope is merged. This would cause duplicate bindings, because the generated
```

Comment:
"and in each the same scope is merged" reads a bit strange, possible to rephrase?